uint8[] is_ended
float64[] current
float64[] reference_feedbackside
uint8[] mode_feedbackside
